modules/ArchiveLibrary/Folder/Repositories/FolderRepository.php

```diff
@@ -223,7 +223,7 @@ public function getFoldersAndFilesByParent(
 
         if ($parentId != null) {
             $filesQuery->where('folder_id', $parentId);
-        }else
+        }elseif ($parentId==null &&!$hasFileFilters )
```

Comment:
The condition `$parentId==null &&!$hasFileFilters` should have proper spacing around operators for better readability. Suggested fix:
```suggestion
        } elseif ($parentId == null && !$hasFileFilters)
```